src/node_crypto.cc

```diff
@@ -5513,10 +5514,14 @@ void TimingSafeEqual(const FunctionCallbackInfo<Value>& args) {
 }
 
 void InitCryptoOnce() {
+  auto env = static_cast<Environment*>(uv_key_get(&thread_local_env));
+  NodeOptions* state = env->options();
   SSL_load_error_strings();
   OPENSSL_no_config();
 
   // --openssl-config=...
+  std::string openssl_config =
```

Comment:
If we change `GetString()` as suggested below, this could be `const std::string&` instead of a copy -- I guess this applies elsewhere, too.